pages/linux/dnf.md

```diff
@@ -1,6 +1,7 @@
 # dnf
 
 > Package management utility for RHEL, Fedora, and CentOS (replaces yum).
+> Some subcommands such as `config-manager` have their own usage documentation.
```

Comment:
And I recall that you had this previously?

```suggestion
> Some subcommands such as `group` and `config-manager` have their own usage documentation.
```

Reply:
Yeah I'll put that back in